documentation/2.0/content/userguide/target_env.md

```diff
@@ -24,7 +33,73 @@ This example would apply the `k8s` target type to the discovery result, and plac
+- `lax` validation will be applied for the resulting model
+- Credentials in the model will be replaced with references to Kubernetes secrets, and a script to create those secrets will be produced
+
+### Using secret credentials in the model
+
+If a target environment is configured to use Kubernetes secrets for credential attribute values, each of those values is replaced with a token using the format `@@SECRET:@@ENV:DOMAIN_UID@@<secret-suffix>:<key>`. For example:
+```yaml
+PasswordEncrypted: '@@SECRET:@@ENV:DOMAIN_UID@@-jdbc-generic1:password@@'
+```
+When a domain is created or updated using a model with these tokens, the environment variable `DOMAIN_UID` should be set to the domain's UID, and secrets with corresponding names should have been created. See [Model Tokens]({{< relref "/concepts/model#model-tokens" >}}) for more details about using secret tokens.
+
+The WebLogic admin credentials use a variation of this token format. For example:
+```yaml
+domainInfo:
+    AdminUserName: '@@SECRET:__weblogic-credentials__:username@@'
+    AdminPassword: '@@SECRET:__weblogic-credentials__:password@@'
+```
+The token `__weblogic-credentials__` allows these attributes to reference secrets in a specific location. The `WDT_MODEL_SECRETS_NAME_DIR_PAIRS` environment variable should be set to associate `__weblogic-credentials__` to this location. For example:
+```shell
+WDT_MODEL_SECRETS_NAME_DIR_PAIRS=__weblogic-credentials__=/etc/my-secrets
+```
+See [Model Tokens]({{< relref "/concepts/model#model-tokens" >}}) for more details about using the `WDT_MODEL_SECRETS_NAME_DIR_PAIRS` environment variable.
+
+In WebLogic Kubernetes Operator environments, the environment variable `DOMAIN_UID` is automatically set from the value in the domain resource file. The variable `WDT_MODEL_SECRETS_NAME_DIR_PAIRS` is automatically set to the directory containing WebLogic admin credentials.
+
+#### The create secrets script
+
+For target environments that use Kubernetes secret tokens, a shell script is created to generate the required secrets. The script is named `create_k8s_secrets.sh`, and is created in the target output directory. The script has commands to create each secret, for example: 
+```shell
+# Update <user> and <password> for weblogic-credentials
+create_paired_k8s_secret weblogic-credentials <user> <password>
+```
+The script should be updated with correct `<user>` and `<password>` values as required. It may be necessary to change the `NAMESPACE` and `DOMAIN_UID` variables at the top of the script if they are different in the target environment.
+
+The script performs a check to determine if any generated secret names are more than 63 characters in length, since that will prevent them from being mounted correctly in the Kubernetes environment. If any secret names exceed this limit, they will need to be shortened in this script, in the model files, and in the domain resource file. Each shortened name should be distinct from other secret names.   
```

Comment:
since -> because (since – don't use to mean for the reason of; use because. Use since only for matters of time, for example, since the last time the program was run.)